From ac0f06e20bdfaca74c8ab786be8ff8fc96b5a689 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 16 Sep 2023 19:52:56 -0700
Subject: [PATCH] cppignore.lnx: Ignore PKCS12 and testLaunch test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These tests fail on Linux

There were 4 failures:
 1: N7CppUnit10TestCallerI11ProcessTestEE.testLaunch
    "rc == 3"
    in "/usr/src/debug/poco/1.12.4-r0/Foundation/testsuite/src/ProcessTest.cpp", line 79
 2: N7CppUnit10TestCallerI11ProcessTestEE.testLaunchRedirectIn
    "rc == 100"
    in "/usr/src/debug/poco/1.12.4-r0/Foundation/testsuite/src/ProcessTest.cpp", line 106
 3: N7CppUnit10TestCallerI11ProcessTestEE.testLaunchRedirectOut
    "s == "Hello, world!""
    in "/usr/src/debug/poco/1.12.4-r0/Foundation/testsuite/src/ProcessTest.cpp", line 134
 4: N7CppUnit10TestCallerI11ProcessTestEE.testLaunchEnv
    "s == "test""
    in "/usr/src/debug/poco/1.12.4-r0/Foundation/testsuite/src/ProcessTest.cpp", line 166

 There were 4 errors:-
   1: N7CppUnit10TestCallerI7EVPTestEE.testRSAEVPKeyFromPKCS12
  │ │ "N4Poco17NotFoundExceptionE:
  Not found: POCO_BASE"
  │ │ in "<unknown>", line -1
   2: N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testFullPKCS12
  │ │ "N4Poco17NotFoundExceptionE:
  Not found: POCO_BASE"
  │ │ in "<unknown>", line -1
   3: N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testCertsOnlyPKCS12
  │ │ "N4Poco17NotFoundExceptionE:
  Not found: POCO_BASE"
  │ │ in "<unknown>", line -1
   4: N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testPEMReadWrite
  │ │ "N4Poco17NotFoundExceptionE:
  Not found: POCO_BASE"
  │ │ in "<unknown>", line -1

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 cppignore.lnx | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/cppignore.lnx
+++ b/cppignore.lnx
@@ -26,3 +26,12 @@ N7CppUnit10TestCallerI9TimerTestEE.testS
 N7CppUnit10TestCallerI9TimerTestEE.testScheduleIntervalClock
 N7CppUnit10TestCallerI9TimerTestEE.testScheduleIntervalTimestamp
 N7CppUnit10TestCallerI9TimerTestEE.testTimer
+N7CppUnit10TestCallerI11ProcessTestEE.testLaunch
+N7CppUnit10TestCallerI11ProcessTestEE.testLaunchRedirectIn
+N7CppUnit10TestCallerI11ProcessTestEE.testLaunchRedirectOut
+N7CppUnit10TestCallerI11ProcessTestEE.testLaunchEnv
+N7CppUnit10TestCallerI7EVPTestEE.testRSAEVPKeyFromPKCS12
+N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testFullPKCS12
+N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testCertsOnlyPKCS12
+N7CppUnit10TestCallerI19PKCS12ContainerTestEE.testPEMReadWrite
+N7CppUnit10TestCallerI11MongoDBTestEE.testArray
